website/docs/docs/running-a-dbt-project/using-the-cli.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Using the CLI"
+---
+
+dbt ships with a Command Line Interface (CLI) for running your dbt project. This way of running dbt a dbt project is free and open source.
```

Comment:
```suggestion
dbt ships with a Command Line Interface (CLI) for running your dbt project. The dbt CLI is free to use and [open source](https://github.com/fishtown-analytics/dbt).
```

My thinking here is just that the open source thing _is_ the software, not the workflow. I don't feel incredibly strongly about this change - let me know if you disagree